install/kubernetes/cilium/templates/hubble/tls-helm/relay-server-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.hubble.enabled .Values.hubble.tls.enabled .Values.hubble.tls.auto.enabled (eq .Values.hubble.tls.auto.method "helm") .Values.hubble.relay.enabled .Values.hubble.relay.tls.server.enabled }}
```

Comment:
Same for this, this is created by default in the `preflight` check. We need to check for `and (.Values.agent (not .Values.preflight.enabled))` here.